wiki/Beatmap/History_of_Loved/en.md

```diff
@@ -44,6 +44,8 @@ After a few rounds of the original style of polling, Toy decided to use a slight
 
 <!-- TODO: Documentation of ~2018 and on -->
 
+In December 2020, development efforts for the official [Project Loved website](https://loved.sh/) began, and shortly after it was launched, merely as an admin page for the coordinators at first. By mid-2021, the website was made available for the rest of the team. Eventually, in August 2021, the Google forms and sheets used for beatmap submissions and viewing were fully deprecated and begun to be hosted on the website instead.
```

Comment:
"December 2020" isn't really accurate, it was on and off for a long time before then too, and it wasn't until sometime 2021 that I sat down and rewrote+completed everything necessary to have people start using it.

I think it may be better to just not mention the development start time (even I don't remember)